.github/workflows/extract_allure_fail_rate.yml

```diff
@@ -0,0 +1,83 @@
+name: Extract Fail Rate from Allure
+
+on:
+  workflow_call:
+    inputs:
+      THRESHOLD:
+        required: true
+        type: number
+    secrets:
```

Comment:
@rgBvtech is this required in this workflow?

Reply:
"THRESHOLD" is required in this workflow. It's passed from the main file (run_behave_tests.yml), in the "ExtractFailRate" job in order to compare it to the fail rate extracted from the allure report. In case you have to change the threshold, you would only have to change it in the main file and not in the workflow steps code ("Extract Fail Rate from Allure").